Updating setup information (#13)
RSE-Sheffield/conda-environments-for-effective-and-reproducible-research: setup.md

## Make sure you have the most recent version

Once Conda exists on your machine, you can update it at any time by running the following command to make sure that you
have the most recent version and patches.

~~~
$ conda update --name base --channel defaults --yes conda
~~~
{: .language-bash}

## Initializing your shell for Conda

Key parts of Conda's functionality require that it interact directly with the shell within which Conda commands are
being invoked as such each shell must be configured to make use of them. The `conda init` command initializes a shell
for use with Conda by making changes to your system that are specific and customized for each shell. Conda supports a
number of different shells and you can run `conda init --help` to see the complete list.

Mac OSX and Linux users may have already been prompted to initialize Conda for your shell when running the installation
script. If so, then you can safely skip this step. 

The following initialises the Bash shell.

~~~
$ conda init bash
~~~
{: .language-bash}

> Windows users can either use the Anaconda Command Prompt or the Anaconda Powershell Prompt which are already initialized

Reviewer: ```suggestion
Windows users should use the Anaconda Powershell Prompt which is already initialized
```

Author: Done, ta. :+1:
